# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/xzgv/gcc-3.3.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/rcfile.c.orig	2001-01-16 03:50:47.000000000 +0200
+++ ./src/rcfile.c	2003-10-19 09:16:03.000000000 +0300
@@ -577,102 +577,102 @@
 {
 printf("xzgv " XZGV_VER
        " - (c) 1999,2000 Russell Marks for improbabledesigns.\n");
-puts("
-usage: xzgv [options] [dir | file ...]
-
-   -a	--auto-hide	automatically hide selector on selecting a picture.
-	--careful-jpeg	enable JPEG `fancy upsampling' (see info file
-			or man page).
-	--delete-single-prompt
-			(normally enabled, use --delete-single-prompt=off to
-			disable) if *disabled*, don't prompt for confirmation
-			when deleting a file.
-	--dither-hicol	use dithering in 15/16-bit to increase apparent
-			colour depth, whatever Imlib's default setting is.
-			You can also use `--dither-hicol=off' to disable
-			this if you normally have Imlib use it.
-	--fast-recursive-update
-			when doing recursive thumbnail update, don't
-			read visible thumbnails for a directory before
-			doing the update (only slightly faster).
-   -f	--fullscreen	use the whole screen for the xzgv window, without
-			even window-manager decorations if possible. (But
-			your wm may not care to trust borderless programs.)
-   -g	--geometry geom
-			use geometry `geom'. For example, `400x300' specifies
-			window size in pixels, `70%x50%' specifies size as
-			percentage of screen width/height, `+100+50' specifies
-			position relative to top-left, and `50%x30%-30%-20%'
-			is left as an exercise for the reader. :-) The default
-			geometry is `92%x85%'.
-			(See info file or man page for more details.)
-   -G	--gamma val	set gamma adjustment to `val'. The default is 1.0, i.e.
-			no adjustment. (See info file or man page for details,
-			and a discussion of gamma issues.)
-   -h	--help		give this usage help.
-	--image-bigness-threshold numpix
-			set the boundary `numpix' above which images are
-			considered `big', and rendered piece-by-piece rather
-			than all-at-once (which is nicer, but harder on
-			memory). Units are number of pixels in image (i.e.
-			width times height), and the default is 2000000 pixels.
-	--interpolate	interpolate between the picture's pixels when
-			scaling up. Usually looks nicer, but it's slow.
-	--mouse-scale-x	if enabled, control-click scales only the X axis -
-			the default is to scale only the Y axis.
-	--revert-orient	(normally enabled, use --revert-orient=off to disable)
-			if *disabled*, orientation (flip/mirror/rotate) state
-			is retained between pictures.
-	--revert-scale	(normally enabled, use --revert-scale=off to disable)
-			if *disabled*, scaling is retained between pictures.
-	--selector-width width
-			set initial/default selector width to `width'. (The
-			units used are pixels, and the normal setting 200.)
-   -T	--show-tagged	show names of tagged files on exit (they're listed
-			to stdout).
-	--show-thumbnail-messages
-			show on the status bar when thumbnails are being read.
-			The status bar must be enabled for the messages to be
-			visible, of course. :-)
-   -k	--skip-parent	for the first directory shown, skip the cursor past
-			`..' (the parent dir). Can be useful when you'd like
-			to immediately use space to `page' through the dir.
-   -o	--sort-order	set initial sorting order used in the selector.
-			Types are `name', `ext', `size', `date' (or `time');
-			only the first char (n/e/s/d/t) need be given.
-			(The default is name order.)
-	--sort-timestamp-type type
-			set timestamp type to use when using time/date sorting
-			order. Types are `mtime' (default), `ctime', and
-			`atime'; only the first char (m/c/a) need be given.
-	--statusbar	show a status bar below the selector; this, for
-			example, says when a picture is being read.
-   -t	--thin-rows	use rows a third the normal height in the selector.
-			This can be very useful on lower-resolution screens,
-			or if you're really interested in filenames, not
-			thumbnails.
-   -v	--version	report version number.
-	--version-gtk	report version of GTK+ being used by xzgv.
-   -z	--zoom		fit pictures in the viewer window, whatever their
-			actual size.
-   -r	--zoom-reduce-only
-			when zooming, only *reduce* pictures to fit; i.e.
-			make big pictures viewable all-at-once while leaving
-			small picures intact.
-
-	dir		start xzgv on a certain directory.
-	file ...	view (only) the file(s) specified.
-
-All options are processed after any ~/.xzgvrc or /etc/xzgv.conf file.
-Most long options (minus `--') can used in either file with e.g. `zoom on'.
-
-On/off settings (such as zoom) are enabled by e.g. `-z' or `--zoom';
-however, the long-option form `--option=off' can be used to disable
-them (needed when they are enabled by default - revert-scale, for
-example - or to override them being enabled in a config file).
-
-(This syntax actually lets you both disable *and* enable options,
-using (for the arg after `=') on/off, y/n, yes/no, or 1/0.)");
+puts("\n"
+"usage: xzgv [options] [dir | file ...]\n"
+"\n"
+"   -a	--auto-hide	automatically hide selector on selecting a picture.\n"
+"	--careful-jpeg	enable JPEG `fancy upsampling' (see info file\n"
+"			or man page).\n"
+"	--delete-single-prompt\n"
+"			(normally enabled, use --delete-single-prompt=off to\n"
+"			disable) if *disabled*, don't prompt for confirmation\n"
+"			when deleting a file.\n"
+"	--dither-hicol	use dithering in 15/16-bit to increase apparent\n"
+"			colour depth, whatever Imlib's default setting is.\n"
+"			You can also use `--dither-hicol=off' to disable\n"
+"			this if you normally have Imlib use it.\n"
+"	--fast-recursive-update\n"
+"			when doing recursive thumbnail update, don't\n"
+"			read visible thumbnails for a directory before\n"
+"			doing the update (only slightly faster).\n"
+"   -f	--fullscreen	use the whole screen for the xzgv window, without\n"
+"			even window-manager decorations if possible. (But\n"
+"			your wm may not care to trust borderless programs.)\n"
+"   -g	--geometry geom\n"
+"			use geometry `geom'. For example, `400x300' specifies\n"
+"			window size in pixels, `70%x50%' specifies size as\n"
+"			percentage of screen width/height, `+100+50' specifies\n"
+"			position relative to top-left, and `50%x30%-30%-20%'\n"
+"			is left as an exercise for the reader. :-) The default\n"
+"			geometry is `92%x85%'.\n"
+"			(See info file or man page for more details.)\n"
+"   -G	--gamma val	set gamma adjustment to `val'. The default is 1.0, i.e.\n"
+"			no adjustment. (See info file or man page for details,\n"
+"			and a discussion of gamma issues.)\n"
+"   -h	--help		give this usage help.\n"
+"	--image-bigness-threshold numpix\n"
+"			set the boundary `numpix' above which images are\n"
+"			considered `big', and rendered piece-by-piece rather\n"
+"			than all-at-once (which is nicer, but harder on\n"
+"			memory). Units are number of pixels in image (i.e.\n"
+"			width times height), and the default is 2000000 pixels.\n"
+"	--interpolate	interpolate between the picture's pixels when\n"
+"			scaling up. Usually looks nicer, but it's slow.\n"
+"	--mouse-scale-x	if enabled, control-click scales only the X axis -\n"
+"			the default is to scale only the Y axis.\n"
+"	--revert-orient	(normally enabled, use --revert-orient=off to disable)\n"
+"			if *disabled*, orientation (flip/mirror/rotate) state\n"
+"			is retained between pictures.\n"
+"	--revert-scale	(normally enabled, use --revert-scale=off to disable)\n"
+"			if *disabled*, scaling is retained between pictures.\n"
+"	--selector-width width\n"
+"			set initial/default selector width to `width'. (The\n"
+"			units used are pixels, and the normal setting 200.)\n"
+"   -T	--show-tagged	show names of tagged files on exit (they're listed\n"
+"			to stdout).\n"
+"	--show-thumbnail-messages\n"
+"			show on the status bar when thumbnails are being read.\n"
+"			The status bar must be enabled for the messages to be\n"
+"			visible, of course. :-)\n"
+"   -k	--skip-parent	for the first directory shown, skip the cursor past\n"
+"			`..' (the parent dir). Can be useful when you'd like\n"
+"			to immediately use space to `page' through the dir.\n"
+"   -o	--sort-order	set initial sorting order used in the selector.\n"
+"			Types are `name', `ext', `size', `date' (or `time');\n"
+"			only the first char (n/e/s/d/t) need be given.\n"
+"			(The default is name order.)\n"
+"	--sort-timestamp-type type\n"
+"			set timestamp type to use when using time/date sorting\n"
+"			order. Types are `mtime' (default), `ctime', and\n"
+"			`atime'; only the first char (m/c/a) need be given.\n"
+"	--statusbar	show a status bar below the selector; this, for\n"
+"			example, says when a picture is being read.\n"
+"   -t	--thin-rows	use rows a third the normal height in the selector.\n"
+"			This can be very useful on lower-resolution screens,\n"
+"			or if you're really interested in filenames, not\n"
+"			thumbnails.\n"
+"   -v	--version	report version number.\n"
+"	--version-gtk	report version of GTK+ being used by xzgv.\n"
+"   -z	--zoom		fit pictures in the viewer window, whatever their\n"
+"			actual size.\n"
+"   -r	--zoom-reduce-only\n"
+"			when zooming, only *reduce* pictures to fit; i.e.\n"
+"			make big pictures viewable all-at-once while leaving\n"
+"			small picures intact.\n"
+"\n"
+"	dir		start xzgv on a certain directory.\n"
+"	file ...	view (only) the file(s) specified.\n"
+"\n"
+"All options are processed after any ~/.xzgvrc or /etc/xzgv.conf file.\n"
+"Most long options (minus `--') can used in either file with e.g. `zoom on'.\n"
+"\n"
+"On/off settings (such as zoom) are enabled by e.g. `-z' or `--zoom';\n"
+"however, the long-option form `--option=off' can be used to disable\n"
+"them (needed when they are enabled by default - revert-scale, for\n"
+"example - or to override them being enabled in a config file).\n"
+"\n"
+"(This syntax actually lets you both disable *and* enable options,\n"
+"using (for the arg after `=') on/off, y/n, yes/no, or 1/0.)");
 
 exit(0);
 }diff -Nru3 trunk-1595/package/jimmy/zgv/gcc-3.3.patch trunk/package/jimmy/zgv/gcc-3.3.patch
